tests/components/ps4/test_config_flow.py

```diff
@@ -37,6 +49,7 @@
     """Test registering an implementation and flow works."""
     flow = ps4.PlayStation4FlowHandler()
     flow.hass = hass
+    manager = flow.hass.config_entries()
```

Comment:
This is not a callable at this point. It's an instance of the `ConfigEntries` class.

No need to use `flow.hass`. `hass` is enough.

Reply:
Alright! Thanks for the tip. Tests passed.